packages/main/src/resolvers/conversation.test.ts

```diff
@@ -361,6 +361,71 @@ describe("when querying conversations", () => {
     })
   })
 
+  it("saves a draft when writing a reply", async () => {
+    const result = await request(
+      `
+      mutation saveDraft($accountId: ID!, $conversationId: ID!, $content: ContentInput!) {
```

Comment:
Good test! You can remove fields from the query that you do not check in the `expect`.